### DISCUSSION TITLE
Is it possible to provide multiple terms?

### DISCUSSION THREAD

**Discussion by VapidLinus** [oramasearch/orama#498] [Q&A] [answered]
Say I want to do a free-text search on all categories, but additionally filter to only results where another term matches a specific category.

The only solution I can think of now is to first do the free-text search, then also do an additional search to filter the specific category, then return the intersection between them.

Say I have a database of games, and the user enters the free-text "candy" but has applied the "genre" filter "roguelike". If I want the result to filter to both of these, what approach should I take?

It seems like this can be done with *filters* but that you then lose the advanced text-matching like thresholds, stemming, stop-words etc.

**Comment by MentalGear** [comment]
Also interested in combining vector with classic tag search.

**Comment by micheleriva** [comment, marked as answer]
This is coming soon!

**Reply by MentalGear** [reply]
May I ask if there will also be an option to combine classic search with vector search; e.g. first search by tag and then filter those results with vector search ?

**Reply by micheleriva** [reply]
We'll release this soon

**Reply by MentalGear** [reply]
PS: I assume the vector search will be integrated into the normal .search interface, unlocking the ability to use functions like field boosting as well?
